VideoGenerator/library/src/main/kotlin/VideoGeneratorStressTest.kt

```diff
@@ -0,0 +1,110 @@
+import org.bytedeco.ffmpeg.global.avcodec
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.lang.Exception
+import java.text.DecimalFormat
+import kotlin.math.log10
+import kotlin.math.pow
+import kotlin.system.measureTimeMillis
+
+class VideoGeneratorStressTest {
+    private lateinit var videoGenerator: VideoGeneratorImpl
+    private val videoPath = "src/resources/testOutput.mkv"
+    private val inputPath = "../example/app/src/androidTest/assets/screen"
+    private val testCodecs =
+        mapOf<Int, Map<String, String>?>(
```

Comment:
Why are the other codecs commented out? did they not conform to our requirements?

Reply:
yes either they did not conform our licence requirements or somehow did not produce a lossless output
JPEG2000 was significantly worse so i commented it out as well
YUV4 was just to compare to raw data